docs/user-guide/playbooks/VPN-with-no-LandingZone.md

```diff
@@ -0,0 +1,352 @@
+# VPN with no Landing Zone
+
+## What
+
+Do you want to try [**binbash Leverage**](/) but you are not willing to transform yet your already existent infra to the [**binbash Leverage Landing Zone**](/try-leverage/) (honoring the [AWS Well Architected Framework](https://docs.aws.amazon.com/wellarchitected/latest/framework/welcome.html))?
+
+With this playbook you will create a VPC with all the benefits **binbash Leverage** [network layer](/user-guide/ref-architecture-aws/features/network/vpc-addressing/) provides.
+
+
+!!! info "If you want to use the Full **binbash Leverage Landing Zone** please visit the [Try Leverage section](/try-leverage/)"
+    This will give you the full power of **binbash Leverage** and the [AWS Well Architected Framework](https://docs.aws.amazon.com/wellarchitected/latest/framework/welcome.html).
+
+## Why
+
+Maybe because you found **binbash Leverage** but you want to try it out first before you convert your base infra.
+
+## How
+ 
+### Objective
+
+We will create a simple VPC (with all its resources) and an will add an EC2 instance to it so we can test it for real.
+
+### What you need:
+
+- an AWS account
+- a user with programmatic access keys and the keys
+- **binbash Leverage**
+
+### Common Process
+
+Ok, just [install binbash Leverage](/try-leverage/local-setup/) and [create the Leverage project](/try-leverage/leverage-project-setup/). In the former, when creating credentials, do it for MANAGEMENT type. 
+Basically:
+```bash
+mkdir project && cd project
+leverage project init
+## edit the yaml file
+leverage project create
+leverage credentials configure --type MANAGEMENT
+```
+
+You end up with this structure:
+```bash
+❯ tree -L 2 .
+.
+├── build.env
+├── config
+│   └── common.tfvars
+├── management
+│   ├── config
+│   ├── global
+│   └── us-east-1
+├── Pipfile
+├── Pipfile.lock
+├── project.yaml
+├── security
+│   ├── config
+│   └── us-east-1
+└── shared
+    ├── config
+    └── us-east-1
+
+```
+
+
+Create a dummy acconunt dir.
+
+Despite we won't create a real account (since there is no Landing Zone), we have to create a dir to hold all the layers we need.
```

Comment:
```suggestion
Despite we won't create a real account (since there is no Landing Zone), we'll have to create a dir to hold all the layers we need.
```